src/env.ts

```diff
@@ -25,3 +25,11 @@ export function region(): string {
 export function publicEndpoint(): string {
   return getEnvironment() === Environment.Staging ? 'myvtexdev.com' : 'myvtex.com'
 }
+
+export function clusterIdDomainInfix(): string {
```

Comment:
What do you think of simply changing the `publicEndpoint()` function to return everything, including this region infix?

Reply:
I thought about it, but most of the time we don't want the infix, because the cookie header will do it and axios lib won't handle the `set-cookie` by cloudfront. But when using `opn` you go to the browser and will get the cookie by going through the redirection process.

Now that you said it, I double checked and I should remove the infix from `sse.ts` and use the cookie header in `createEventSource`